src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,130 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exeption.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final Logger logger = LogManager.getLogger(ManufacturerDaoImpl.class);
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        logger.info("Method get was called with id: " + id);
+        String getManufacturersRequest = "SELECT * FROM manufacturers WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getManufacturer =
+                        connection.prepareStatement(getManufacturersRequest)) {
+            getManufacturer.setObject(1, id);
+            ResultSet resultSet = getManufacturer.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(parseManufacturer(resultSet));
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can`t get manufacturer by id from db "
+                    + id, throwables);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String getAllManufacturersRequest = "SELECT * FROM manufacturers WHERE is_deleted = false;";
+        List<Manufacturer> allManufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllManufacturersStatement = connection.createStatement()) {
```

Comment:
It can be named simpler. We already see the context in the method signature 
```suggestion
        String query = "SELECT * FROM manufacturers WHERE is_deleted = false;";
        List<Manufacturer> allManufacturers = new ArrayList<>();
        try (Connection connection = ConnectionUtil.getConnection();
                Statement statement = connection.createStatement()) {
```